.travis.yml

```diff
@@ -0,0 +1,91 @@
+language: python
+sudo: required
+
+notifications:
+  email: false
+
+services:
+  - docker
+
+python:
+  - "2.6"
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+
+env:
+  global:
+    - ICU_VERSION="58.2"
+    - PYICU_LIBRARIES="icui18n:icuuc:icudata"
+    - PYICU_INCLUDES="/tmp/icu4c/include"
+    - PYICU_LFLAGS="-L/tmp/icu4c/lib"
+  matrix:
+    - ARCH=x86_64 PRE_CMD=
+    - ARCH=i686 PRE_CMD=linux32
+
+matrix:
+  include:
+    - python: "2.6"
+      env: ARCH=x86_64 PRE_CMD= UCS=u
+      services:
+        - docker
+    - python: "2.6"
+      env: ARCH=i686 PRE_CMD= UCS=u
+      services:
+        - docker
+    - python: "2.7"
+      env: ARCH=x86_64 PRE_CMD= UCS=u
+      services:
+        - docker
+    - python: "2.7"
+      env: ARCH=i686 PRE_CMD= UCS=u
+      services:
+        - docker
+
+cache:
+  directories:
+    - /tmp/icu4c
+
+before_install:
+  - docker pull quay.io/pypa/manylinux1_$ARCH
+  - mkdir -p /tmp/icu4c
+  - export ICU_URL="https://downloads.sourceforge.net/project/icu/ICU4C/$ICU_VERSION/icu4c-${ICU_VERSION/./_}-src.tgz"
+  - wget -nv -O /tmp/icu4c/icu4c.tgz "$ICU_URL"
+  - tar -xzf /tmp/icu4c/icu4c.tgz -C /tmp/icu4c
+  - >-
+      docker run --rm
+      -v /tmp/icu4c:/tmp/icu4c
+      quay.io/pypa/manylinux1_$ARCH
+      $PRE_CMD bash -c
+      "cd /tmp/icu4c/icu/source && ./configure --prefix=/tmp/icu4c && make && make install"
+
+install:
+  - >-
+      docker run --rm
+      -v /tmp/icu4c:/tmp/icu4c
+      -v `pwd`:/io
+      -e ICU_VERSION -e PYICU_LIBRARIES -e PYICU_INCLUDES -e PYICU_LFLAGS -e TRAVIS_PYTHON_VERSION
+      quay.io/pypa/manylinux1_$ARCH
+      $PRE_CMD bash -c
+      "/opt/python/cp${TRAVIS_PYTHON_VERSION/./}-cp${TRAVIS_PYTHON_VERSION/./}m$UCS/bin/pip wheel /io -w /wheelhouse &&
+      LD_LIBRARY_PATH=/tmp/icu4c/lib auditwheel repair /wheelhouse/* -w /io/wheelhouse"
+
+script:
+  - >-
+      docker run --rm
+      -v `pwd`:/io
+      quay.io/pypa/manylinux1_$ARCH
+      $PRE_CMD bash -c
+      "/opt/python/cp${TRAVIS_PYTHON_VERSION/./}-cp${TRAVIS_PYTHON_VERSION/./}m$UCS/bin/pip install --pre --find-links /io/wheelhouse pyicu pytest six &&
+      /opt/python/cp${TRAVIS_PYTHON_VERSION/./}-cp${TRAVIS_PYTHON_VERSION/./}m$UCS/bin/py.test /io/test"
+
+deploy:
+  provider: releases
+  api_key:
+    secure: neU15Xmkc4H/pzNFQOF5JgF5RQxAOkPNY4aGaBMvmEVyFcfOxYQOJoKCwzm0jpqXe2Ce+7tsmbQiCrkQoZrWcIyg2tX0iW+Mi1qr86iQMUuTXbVHxsZxy//IkHjVb20a8jTLqYdCM7s1bqWsi5inR11hD9bDVXFGqId5qIqlI24=
```

Comment:
This is for uploading to my fork. You'd want to change this to your encrypted oauth token, and possibly enable automatic PyPI uploads as well.